Improve the API docs, module MD, and package MD (#138)
ballerina-platform/module-ballerina-xslt: xslt-ballerina/Module.md

## Module Overview

> This module provides a function to transform the XML content to another XML/HTML/plain text using XSL transformations.

Reviewer: ```suggestion
This module provides an API to transform the XML content to another XML/HTML/plain text using XSL transformations.
```
